core/src/test/java/net/iakovlev/timeshape/TimeZoneEngineSerializationTest.java

```diff
@@ -0,0 +1,95 @@
+package net.iakovlev.timeshape;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.io.IOException;
+import java.io.BufferedOutputStream;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+import java.time.ZoneId;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.io.File;
+/*
+import com.esri.core.geometry.Envelope;
+import com.github.luben.zstd.ZstdInputStream;
+import net.iakovlev.timeshape.proto.Geojson;
+import org.apache.commons.compress.archivers.tar.TarArchiveEntry;
+import org.apache.commons.compress.archivers.tar.TarArchiveInputStream;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.time.ZoneId;
+import java.util.*;
+import java.util.function.Consumer;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+*/
```

Comment:
Please clean up commented out code.

Reply:
Oops. Commented out imports are removed.
Added some text about serialization to Readme.
Took the liberty of adding my app to the list of users.